docs/wfirst.rst

```diff
@@ -0,0 +1,96 @@
+******************
+WebbPSF for WFIRST
+******************
+
+
+
+.. figure:: ./wfirst_figures/webbpsf-wfirst_page_header.png
+   :align: center
+   :alt: Sample PSFs for the filters in the WFIRST WFI.
+
+   Sample PSFs for the filters in the WFIRST WFI. Angular scale in arcseconds, log-scaled intensity.
+
+Introduction
+============
+
+WebbPSF provides a framework for JWST instrument PSF calculations that is easily extensible to other instruments and observatories. The :py:mod:`webbpsf.wfirst` module was developed to enable simulation of WFIRST's Wide Field Instrument (WFI) based on reference information from Goddard Space Flight Center.
+
```

Comment:
Let's add here a hyperlink to http://wfirst.gsfc.nasa.gov/science/Instrument_Reference_Information.html to be specific.